pdr_backend/pdr_dashboard/util/db.py

```diff
@@ -1,20 +1,61 @@
 import logging
+from copy import deepcopy
+from typing import Any, Dict, List, Optional, Tuple, Union
 
-from typing import Union, List
 from enforce_typing import enforce_types
 
 from pdr_backend.lake.duckdb_data_store import DuckDBDataStore
 from pdr_backend.lake.payout import Payout
 from pdr_backend.lake.prediction import Prediction
 from pdr_backend.lake.subscription import Subscription
+from pdr_backend.pdr_dashboard.util.data import (
+    col_to_human,
+    filter_objects_by_field,
+    get_feed_column_ids,
+    get_feeds_stat_with_contract,
+    get_feeds_subscription_stat_with_contract,
+)
+from pdr_backend.pdr_dashboard.util.format import format_dict, format_table
+from pdr_backend.pdr_dashboard.util.prices import (
+    calculate_tx_gas_fee_cost_in_OCEAN,
+    fetch_token_prices,
+)
 from pdr_backend.util.constants_opf_addrs import get_opf_addresses
 
 logger = logging.getLogger("predictoor_dashboard_utils")
 
 
+# pylint: disable=too-many-instance-attributes
 class DBGetter:
-    def __init__(self, lake_dir: str):
-        self.lake_dir = lake_dir
+    def __init__(self, ppss):
+        self.lake_dir = ppss.lake_ss.lake_dir
+        self.network_name = ppss.web3_pp.network
+
+        # fetch token prices
+        self.fee_cost = calculate_tx_gas_fee_cost_in_OCEAN(
+            ppss.web3_pp,
+            "0x18f54cc21b7a2fdd011bea06bba7801b280e3151",
+            fetch_token_prices(),
+        )
+
+        # initial data loaded from database
+        self.feeds_payout_stats = self._init_feed_payouts_stats()
+        self.feeds_subscriptions = self._init_feed_subscription_stats()
+        self.predictoors_data = self._init_predictoor_payouts_stats()
+        self.feeds_data = self._init_feeds_data()
+
+        # initial data formatting for tables, columns and raw data
+        self.feed_cols, self.feed_table_data, self.feed_data_raw = (
+            self._format_for_feeds_table()
+        )
+        self.predictoor_cols, self.predictoor_table_data, self.raw_predictoor_data = (
```

Comment:
Could you make the variable names more constant? For example: raw_predictoor_data and feed_data_raw make them: raw_predictoor_data and raw_feed_data or the other way around.

Reply:
I could, yes. I kept some original names but I can make it even more uniform 👍